webcompat/views.py

```diff
@@ -147,6 +147,7 @@ def index():
 
 
 @app.route('/issues')
+@cache_policy(private=True, uri_max_age=0, must_revalidate=True)
```

Comment:
Just so I understand: here and for the other endpoints with this decorator, by setting `uri_max_age=0`, you're telling the user agent that it should revalidate the resource always?

Do we plan on using a longer (i.e., the day default) max age for other resources, in a follow up bug perhaps?